.ci/create_vm.gitlab-ci.yml

```diff
@@ -0,0 +1,161 @@
+#
+# Create VM with RHEL9
+#
+create:rhel9:vm:
+  stage: create vm
+  tags:
+    - docker-throttled
+  script:
+    - echo "TEST HAS STARTED"
+    - apt-get update -y
+    - apt-get install curl -y
```

Comment:
on rhel 9 it's yum, not apt

Reply:
In this step we are not on RHEL9 yet, VM with RHEL is not created yet. We are on docker executor which is ubuntu16.04 and we're installing curl on this ubuntu image to send request to create RHEL9 VM.